arangod/RestHandler/RestAnalyzerHandler.h

```diff
@@ -31,7 +31,7 @@ namespace iresearch {
 
 class IResearchAnalyzerFeature; // forward declaration
 
-class RestAnalyzerHandler final: public RestVocbaseBaseHandler {
```

Comment:
why not final ?

Reply:
because RestHandlerCreator is a template class that extends its template argument, in this case its template argument is RestAnalyzerHandler
@see GeneralServerfeature.cpp